tomls/omnibus-base-sepolia-andromeda/spot/wsteth.toml

```
@@ -0,0 +1,92 @@
+[var.wsteth_spot_settings]
+synth_wsteth_max_collateral_amount = "<%= parseEther(String(1_750)) %>"
+synth_wsteth_skew_scale = "<%= parseEther(String(350_000)) %>"
+
+[invoke.SpotMarketProxy_createSynth_wsteth]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "owner"
+func = "createSynth"
+args = ["Synthetic Lido Wrapped Staked ETH", "sWSTETH", "<%= settings.owner %>"]
```

```suggestion
args = ["Synthetic Lido Wrapped Staked ETH", "wstETH", "<%= settings.owner %>"]
```

also technically token name is `Wrapped liquid staked Ether 2.0` from contract https://etherscan.io/token/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0#readContract

---

No worries sir will add the 2.0 😅  although what is 1.0

---

Well that almost caused an issue @noisekit , thankfully fixed 😅  missing `s` sir